src/commands/es/migrate.ts

```diff
@@ -0,0 +1,134 @@
+import { flags } from '@oclif/command'
+import { Client } from '@elastic/elasticsearch'
+import cli from 'cli-ux'
+import chalk from 'chalk'
+import emoji from 'node-emoji'
+
+import { Kommand } from '../../common'
+
+export default class EsMigrate extends Kommand {
+    static initSdk = false
+
+    static description = 'Migrate all the index from one Elasticsearch server to another'
+
+    static flags = {
+        help: flags.help(),
+        src: flags.string({
+            description: 'Source Elasticsearch server URL (',
+            env: 'KUZZLE_ES_SRC',
+            required: true
+        }),
+        dest: flags.string({
+            description: 'Destination Elasticsearch server URL',
+            env: 'KUZZLE_ES_DEST',
+            required: true
+        }),
+        reset: flags.boolean({
+            description: 'Reset destination Elasticsearch server',
+            default: false,
+        }),
+        limit: flags.integer({
+            description: 'Document transfer limit',
+            default: 1000,
+        }),
+        force: flags.boolean({
+            description: 'Force migration and skip confirmation interactive prompts',
+            default: false,
+        }),
+    }
+
+    static args = []
+
+    private src: any
+
+    private dest: any
+
+    private async migrateMappings(index: string) {
+        const { body } = await this.src.indices.getMapping({ index, format: 'json' })
+        const mappings = body[index]
+        await this.dest.indices.create({ index, body: mappings })
+    }
+
+    private async migrateData(index: string, limit: number) {
+        const queue = []
+        // Quick scroll duration estimation based on number of documents per hit
+        const scrollDuration = 5 * (limit % 1000) > 0 ? `${5 * (limit % 1000)}s` : '5s'
```

Comment:
I don't understand the purpose of this line since the value of `scrollDuration` wont grow linearly with the value of `limit`.   
On the opposite, a small `limit` like `103` will give me `515s` of `scrollDuration`